src/GitTools.Core/GitTools.Core.NETSTANDARD/project.json

```diff
@@ -0,0 +1,40 @@
+﻿{
+  "supports": {},
+  "dependencies": {
+    "LibGit2Sharp-dotnet": "1.0.0",
+    "LibGit2Sharp.NativeBinaries-dotnet": "[1.0.137]",
```

Comment:
Currently uses my fork of LibGit2Sharp with .net core support. Will update this to the official packages once they are available.